startupscript/gcp/post-startup-hook.sh

```diff
@@ -16,7 +16,7 @@
 INSTANCE_REGION="$(get_instance_region)"
 
 # Check if gcloud is logged in and set the region if it is
-ACCOUNT="$(gcloud config get-value account 2>/dev/null | tr -d '[:space:]')"
+ACCOUNT="$(${RUN_AS_LOGIN_USER} "gcloud config get-value account 2>/dev/null" | tr -d '[:space:]')"
```

Comment:
The first gcloud command executed will permission the `.gcloud` directory with the caller as owner. We don't want that to be root.